Ghidra/Features/Base/src/main/java/ghidra/app/plugin/core/label/EditLabelAction.java

```diff
@@ -64,6 +65,12 @@ protected boolean isEnabledForContext(ListingActionContext context) {
 			if (symbol.isExternal()) {
 				return false;
 			}
+			if (symbol.getSymbolType() == SymbolType.FUNCTION) {
+				if (LabelMgrPlugin.getDataComponent(context) != null) {
```

Comment:
We might want to remove this check entirely. This would eliminate all the competition between the "Rename Function" and "Edit Label" actions.

Reply:
I did initially have a concern about it which is why I added the data check but I'll admit that I don't remember what it was. It makes sense to me to let the "Rename Function" action handle all function symbol types.

Reply:
I have to add a check to see if the context location is the label of the function and return true in that case. The data check was initially a poor solution to that problem.